README.md

```diff
@@ -1,85 +1,67 @@
+
+### Representation on the Board of Directors
+The Individual Members are represented on the Board of Directors by 2 Individual Directors 
+with staggared two year terms. A representivite is elected in January of each year with
+no limitations to the number of terms served.
+
+#### Current Individual Directors
+<!-- Senior Director always listed first. Upon re-election, they become the Junior again.-->
+<table>
+  <tr>
+    <td>
+      <a href="//github.com/ashleygwilliams">
+        <img width=200 src="https://avatars.githubusercontent.com/u/1163554?s=200" alt="@ashleygwilliams">
+      </a>
+      <br>
+      **Ashley Williams**<br>
+      <a href="//github.com/ashleygwilliams">@ashleygwilliams</a><br>
+      <sup>Jan 2016 - Jan 2018</sup>
+    </td>
+    <td>
+      <a href="//github.com/williamkapke">
+        <img width=200 src="https://avatars.githubusercontent.com/u/739813?s=200" alt="@williamkapke">
+      </a>
+      <br>
+      **William Kapke**<br>
+      <a href="//github.com/williamkapke">@williamkapke</a><br>
+      <sup>Jan 2017 - Jan 2019</sup>
+    </td>
+  </tr>
+<table>
+
+Along with their regular Board duties, the Individual Directors use 
+this repository to communitcate with the Membership.
+
+#### Previous Individual Directors
+- Feross Aboukhadijeh [@feross](https://github.com/feross) • Jan 2016 - Jan 2017.
+
+### Registration and Cost
+Anyone with a GitHub account may become an Individual Member. Visit 
+https://identity.linuxfoundation.org/pid/99 to register.
+
+The cost of Individual Membership is:
+- Node.JS Project Contributors: [__free__][1]*
 - Students: __$25/yr__
 - General Population: __$100/yr__
 
-*This is an issue on a Private repo that is only accessible to GitHub users
-who are on the [`Members` team][8] on the [`nodejs` GitHub organization][9].
-Users are added to that team when they contribute consistently to Node.js
-projects or participate in one of the [Working Groups][10].
-
-## Registration
-
-Visit [https://identity.linuxfoundation.org/pid/99][3]. Be sure to login via Github.
-
-# Questions? Comments?
-
-[Please file an issue.][6]
-
-## Representation on the Board of Directors
-
-The Board meets every month to approve resolutions and discuss
-Node.js Foundation administrative matters. This includes legal
-considerations, budgeting and approving Foundation-led
-conferences and other initiatives. Technical governance is
-overseen by the [TSC][7], not the Board of Directors.
-
-### Term Lengths
+<sup>* links to an issue on a Private repo that is only accessible to GitHub users
+who are on the [`Members` team](https://github.com/orgs/nodejs/teams/members) of
+the `nodejs` GitHub organization. Users are added to the team (by Collaborators)
+when they contribute consistently to a Node.js Foundation project.</sup>
 
-The standard term length for those elected by the individual
-membership is 2 years, with an election each year to select
-a new representative for a new term.
+## Questions? Comments?
```

Comment:
It can be useful to add some direction to these types of sections, e.g. "Interested in running for this seat? Want to raise a concern to the board? Looking for documentation?"

Usually adding stuff like that means you are more likely to get actionable feedback